app.py

```diff
@@ -67,12 +66,10 @@
 def setup_cira():
     global cira
 
-    # determine the location of the pre-trained models
-    dotenv.load_dotenv()
-    model_env_suffix = '_DEV' if ('DEV_CONTAINER' in os.environ) else ''
-    model_classification = os.environ[f'MODEL_CLASSIFICATION{model_env_suffix}']
-    model_labeling = os.environ[f'MODEL_LABELING{model_env_suffix}']
-
+    model_classification = model_locator.classification()
+    model_labeling = model_locator.labeling()
+    print(f'Classification model path: {model_classification}')
```

Comment:
Are the print statements advisable in production code?

Reply:
In general no, but in this case yes. Since we had multiple occasions where it was not clear whether the `_DEV` version of an env was used.

Reply:
Or do you use a logger anywhere?

Reply:
No, not yet. We could take this issue up in the future.